testing/regress/ecl/stresstext.ecl

```diff
@@ -23,6 +23,10 @@
 //version multiPart=false,variant='inplace_lzw'
 //version multiPart=false,variant='inplace_lz4hc'
 
+#option('roxie:minPayloadSize', 10000)
```

Comment:
I don't think these changes should be released.

Reply:
I will comment them out.